packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3136,6 +3137,9 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     bool scopesRoute,
     bool namesRoute,
     bool hidden,
+    bool image,
+    bool liveRegion,
+    bool isSwitch,
```

Comment:
this argument is ignored. was that a mistake?

Reply:
yup

Reply:
i have a PR out to remove it